api/types.go

```diff
@@ -71,6 +71,10 @@ func (t Time) Equal(other Time) bool {
 	return time.Time(t).Equal(time.Time(other))
 }
 
+func (t Time) Expired(expiry Time) bool {
+	return time.Time(t).UTC().Before(time.Time(expiry).UTC())
+}
```

Comment:
This `api.Time` type could be used for all kinds of things, not necessarily only for expiry times. 

Since this method is only used in tests, I think it would be better to keep this logic in the test rather than expose it as a method.